config/environments/production.rb

```diff
@@ -49,5 +49,9 @@
         Sentry.capture_exception(exception)
       },
     }
+
+    cache_store_config.merge({ password: ENV['LAGO_REDIS_CACHE_PASSWORD'] }) if ENV['LAGO_REDIS_CACHE_PASSWORD'].present?
```

Comment:
Layout/LineLength: Line is too long. [121/120] (https://rubystyle.guide#max-line-length)